content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -729,6 +730,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   assignments for different components in Kubernetes. See
   [Control Topology Management Policies on a node](/docs/tasks/administer-cluster/topology-manager/).
 - `UserNamespacesStatelessPodsSupport`: Enable user namespace support for stateless Pods.
+- `ValidatingAdmissionPolicy`: Enable support for CEL validations be used in Admission Control.
```

Comment:
(nit)
Please hyperlink to `/docs/reference/access-authn-authz/validating-admission-policy/` here.